CrossPlatformDesktopProject/CrossPlatformDesktopProject/NPC/Boss/Boss.cs

```diff
@@ -12,7 +12,7 @@ namespace CrossPlatformDesktopProject.NPC
 {
     class Boss : INpc
     {
-        private INpcState currentState;
+        public INpcState currentState;
```

Comment:
Same with Bat, as well as with the `xPos` and `yPos` from line 16 of this file

Reply:
Yes, same as with Bat I need those to be public